etc/config/fortran.amazon.properties

```diff
@@ -129,6 +129,15 @@ compiler.fppc64g8.exe=/opt/compiler-explorer/powerpc64/gcc-at12/powerpc64-unknow
 compiler.fppc64g8.name=power64 AT12.0
 compiler.fppc64g8.semver=(snapshot)
 
+###############################
+# ARM HPC
+group.armhpc.compilers=armhpc-ac193a64
```

Comment:
&armhpc needs to be added to group.cross.compilers a little bit further up (line 94)